pom.xml

```diff
@@ -67,6 +67,7 @@
     <plugins>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M5</version>
```

Comment:
I extracted this to a single commit 416ca75f65134588371abffdd7c42d11ccb3db02 because I like to have one commit per change so that its purpose can be explained in the commit message.

Reply:
sure no problem, I probably should have done that in the first place anyway.